source/_includes/installation/operating_system.md

```diff
@@ -91,6 +91,18 @@ To use this method, follow the steps described in the procedure below: [Write th
 
 - To use this method, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
 
+<div class='note'>
```

Comment:
Let's not add a big note block for this. I think it is more fitting to put this in a details block instead.